thrift/thrift0.13/src/main/java/com/linecorp/armeria/server/thrift/ThriftCallService.java

```diff
@@ -63,38 +57,20 @@ public void onError(Exception e) {
     };
 
     /**
-     * Creates a new {@link ThriftCallService} with the specified service implementation.
-     *
-     * @param implementation an implementation of {@code *.Iface} or {@code *.AsyncIface} service interface
-     *                       generated by the Apache Thrift compiler
+     * Creates a new instance of {@link ThriftCallServiceBuilder} which can build
+     * an instance of {@link ThriftCallService} fluently.
      */
-    public static ThriftCallService of(Object implementation) {
-        requireNonNull(implementation, "implementation");
-        return new ThriftCallService(ImmutableMap.of("", ImmutableList.of(implementation)));
-    }
```

Comment:
Shall we keep this method because otherwise we would break backward compatibility? We could reimplement this method like this:

```
    return builder().implementation(implementation).builder();
```

Reply:
I applied it. thank you!